src/Redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import missionsSlice from './Missions/missionsSlice';
+import rocketsSlice from './Rockets/rocketsSlice';
+
+const rootReducer = combineReducers({
+  rockets: rocketsSlice,
+  missions: missionsSlice,
+});
+
+const store = configureStore({ reducer: rootReducer });
+
+export default store;
```

Comment:
- [ ] Kindly use the `redux-logger` package for inspecting the store. Kindly install it and use it for the project as stated in [the requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#config--basic-setup).

  - Install with the command `npm i redux-logger`
  - Modify the store to include it:
    
    ```
    import { configureStore, combineReducers } from '@reduxjs/toolkit';
    import logger from 'redux-logger';
    import missionsSlice from './Missions/missionsSlice';
    import rocketsSlice from './Rockets/rocketsSlice';
    
    const rootReducer = combineReducers({
      rockets: rocketsSlice,
      missions: missionsSlice,
    });
    
    const store = configureStore({
      reducer: rootReducer,
      middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger)
    });
    
    export default store;
    ```

  - Open the browser console to view the store.